frontend/local/src/containers/patients/detail/index.js

```diff
@@ -76,22 +76,22 @@ class PatientDetail extends React.Component {
                                 waitlistItem.vitalSigns.height && (
                                     <div className="col-sm-4">
                                         <h5>Height</h5>
-                                        {waitlistItem.vitalSigns.height}cm
+                                        {waitlistItem.vitalSigns.height.value} cm
                                     </div>
                                 )}
                             {waitlistItem.vitalSigns &&
                                 waitlistItem.vitalSigns.weight && (
                                     <div className="col-sm-4">
                                         <h5>Weight</h5>
-                                        {waitlistItem.vitalSigns.weight}kg
+                                        {waitlistItem.vitalSigns.weight.value} kg
                                     </div>
                                 )}
                             {waitlistItem.vitalSigns &&
                                 waitlistItem.vitalSigns.height &&
                                 waitlistItem.vitalSigns.weight && (
                                     <div className="col-sm-4">
                                         <h5>BMI</h5>
-                                        {round(waitlistItem.vitalSigns.weight / waitlistItem.vitalSigns.height / waitlistItem.vitalSigns.height * 10000, 2)}
+                                        {round(waitlistItem.vitalSigns.weight.value / waitlistItem.vitalSigns.height.value / waitlistItem.vitalSigns.height.value * 10000, 2)}
```

Comment:
Would it make sense to change bmi into "normal value". We could set it after form submit if both height and weight are defined.

Reply:
ok